radio/config.py

```diff
@@ -1,3 +1,4 @@
 config = dict(
     COMM_SECURITY_CODE=0x11223344,
+    COMM_UPLINK_CALLSIGN="SQ5PUI"

```

Comment:
i'd recommend getting this from an environment varriable instead of configuration file.  This way configuration for both gses can be the same.

Reply:
Actually you can't share the config file on GitHub so it will be created on gs side... It's not clear for me to put all important stuff to env var

Reply:
wyniki ankiety na slacku wskazują na to, że callsign powinien pozostać w configu @zaq32

Reply:
Ten config i tak jest niepotrzebny, bo radio_comm powinno korzystać z configa, który jest poza repo, więc tu powinien być posty callsign

Reply:
wrrr

Reply:
@Novakov sprawdź teraz :)

Reply:
Teraz lepiej :)